.travis.yml

```diff
@@ -6,6 +10,9 @@ env:
     - PANTS_CONFIG_FILES="${TRAVIS_BUILD_DIR}/pants.travis-ci.ini"
     - PYTEST_PASSTHRU_ARGS="-v --duration=3"
     - LC_ALL="en_US.UTF-8"
+    - BOOTSTRAPPED_PEX_BUCKET=ci-public.pantsbuild.org
+    - BOOTSTRAPPED_PEX_KEY=${TRAVIS_BUILD_NUMBER}/${TRAVIS_BUILD_ID}/pants.pex
```

Comment:
Worth including the SHA as an additional stopgap?

Reply:
Not sure if that has much value in a world-writeable bucket, since whoever overwrites the file can also overwrite the sha?